### ISSUE TITLE
reimplement mxSave/mxRestore using mxComputeCheckpoint

### ISSUE THREAD

**Issue by jpritikin** [OpenMx/OpenMx#230]
We can save and restore much more of the information than the current implementation.

**Comment by mcneale** [comment]
True.  The Hessian for one, standard errors, CIs and a bunch of fit statistics for another few.  The problem with the latter is that some may not be available, but NA may be fine.  Do we have a definitive (wish) list?

**Comment by jpritikin** [comment]
Actually `mxComputeCheckpoint` doesn't help because `mxSave` is invoked from the R frontend. All the code to save various things would need a fresh implementation in R. The one place where `mxComputeCheckpoint` could help is just to ensure that the output format is the same for the frontend and backend. So `mxRestore` should work equally well using a file generated by `mxSave` or `mxComputeCheckpoint`.

**Comment by jpritikin** [comment]
Decision from Aug 30 meeting, go ahead and

- implement `mxRestore` in terms of `mxRun`
- do a matching `mxSave` implementation
